tmt/base.py

```diff
@@ -1499,6 +1515,14 @@ def stories(self, keys=None, names=None, filters=None, conditions=None,
         links = (links or []) + list(Story._opt('links', []))
         excludes = (excludes or []) + list(Story._opt('exclude', []))
 
+        # Get post filter to support option '--enabled/--disabled'
+        if Story._opt('enabled'):
+            post_filters = ['enabled:true']
```

Comment:
What about just extending the `filters` list?

```suggestion
            filters.append('enabled:true')
```

There's no additional processing of it so this will work nicely and we can also get rid of the `else` part below.

Reply:
@psss and @lukaszachy, `post_filters` is removed and use `filters.append()` instead.

Reply:
Thanks!